app/javascript/mastodon/locales/defaultMessages.json

```diff
@@ -968,27 +968,6 @@
   {
     "descriptors": [
       {
-        "defaultMessage": "New report",
-        "id": "report.heading"
-      },
-      {
-        "defaultMessage": "Additional comments",
-        "id": "report.placeholder"
-      },
-      {
-        "defaultMessage": "Submit",
-        "id": "report.submit"
-      },
-      {
-        "defaultMessage": "Reporting",
-        "id": "report.target"
-      }
-    ],
-    "path": "app/javascript/mastodon/features/report/index.json"
-  },
```

Comment:
That file has been deleted, but the keys remain in the report modal. I cannot reproduce the deletion of all these report keys when I run yarn run manage:translations on master.